### DISCUSSION TITLE
Stackoverflow 2021年技術問卷調查

### DISCUSSION THREAD

**Discussion by chia-kai-chang** [1101-GS4719-C/Python-Community-GS4719-C#563] [General]
相當精彩的問卷調查分析，有熱門技術排序、開發人員常用工具、職稱、薪水等各種排序資料與交叉比對資料，如果你想了解軟體圈生態，一定要細細從這些圖表中，體會世界的技術脈動。

> 網址：https://insights.stackoverflow.com/survey/2021

![image](https://user-images.githubusercontent.com/1768891/165141267-e9f827b5-f242-4ba8-b55c-749d10826cc4.png)

![image](https://user-images.githubusercontent.com/1768891/165141316-c46c9598-143f-4042-a57c-2239efbed46c.png)

### Database交叉比對 (僅示範，還有更多比較)

![image](https://user-images.githubusercontent.com/1768891/165141505-02c5f943-1917-4e5b-9be6-f4a96f4a5593.png)

**Comment by JSHT** [comment]
沒想到 oracle cloud 占比這麼少，他的 free tier 方案提供的運算資源算多ㄝ。

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，讓我知道世界的技術脈動。
